libvast/vast/fbs/partition.fbs

```diff
@@ -18,22 +18,38 @@ table QualifiedValueIndex {
   index: ValueIndex;
 }
 
+table TypeIds {
+  /// The type name.
+  name: string;
+
+  /// The serialized `vast::ids`.
+  ids: [ubyte];
+}
+
 /// The persistent state of the meta index.
+//  TODO: Add a `Version` field.
 table Partition {
   /// The UUID of this partition.
   uuid: UUID;
 
+  /// A human-readable name for this partition
+  name: string;
```

Comment:
Isn't the partition name simply the UUID?

Reply:
Right, removed.